build/linux/debian/checkout-shlibdeps.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+sudo apt-get update
+sudo apt-get install wget
```

Comment:
All the linux agents have `curl` installed, please use it instead.

Reply:
In that case, considering it's a single statement, I can inline it into the TS file.

Reply:
Sure, that is also fine.